src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/utils/SecurityUtils.java

```diff
@@ -39,6 +39,15 @@ public static String getClientId() {
         }
     }
 
+    public static String getIdentityProvider() {
+        try {
+            return ((KeycloakPrincipal) SecurityContextHolder.getContext().getAuthentication().getPrincipal())
```

Comment:
I feel like we have a duplicate here
Can we have a private method `getCustomClaimValue(String key)` ?

Reply:
Yes it should